### ISSUE TITLE
UK pound symbol

### ISSUE THREAD

**Issue by johnroberts2k** [the-raspberry-pi-guy/lcd#29]
I'm trying out this library to get a 16x2 LCD to work on my Pi. I need to display the uk pound £ sign. Is this possible to do with this library?

**Comment by cgomesu** [comment]
yes, it is possible.  the pound sign is `A3` in hex (`unichr(163)`) but I think you will have a hard time trying to pass that via the usual write function, namely `lcd_display_string()`.  instead, check the [custom characters usage](https://github.com/the-raspberry-pi-guy/lcd#custom-characters) that @juvus added to the code.  you can write pretty much anything that way (but there's a limit to how many custom characters you can hold in memory).
